core/server/worker/src/main/java/alluxio/worker/block/MonoBlockStore.java

```diff
@@ -264,8 +281,68 @@ public void requestSpace(long sessionId, long blockId, long additionalBytes)
     mLocalBlockStore.requestSpace(sessionId, blockId, additionalBytes);
   }
 
+  @Override
+  public List<BlockStatus> load(List<Block> blocks, String tag, OptionalInt bandwidth) {
+    ArrayList<CompletableFuture<BlockStatus>> futures = new ArrayList<>();
+    for (Block block : blocks) {
+      CompletableFuture<Void> future = CompletableFuture.runAsync(() -> {
+        try {
+          loadInternal(block.getBlockId(), block.getBlockSize(), block.getMountId(),
+              block.getUfsPath(), tag, bandwidth, block.getOffsetInFile());
+        } catch (Exception e) {
+          throw AlluxioRuntimeException.from(e);
+        }
+      }, GrpcExecutors.BLOCK_READER_EXECUTOR);
+      CompletableFuture<BlockStatus> exceptionally = future.handle((res, throwable) -> {
+        if (throwable != null) {
+          return BlockStatus.newBuilder().setBlock(block)
+              .setCode(AlluxioRuntimeException.from(throwable).getStatus().getCode().value())
+              .setMessage(throwable.getMessage()).setRetryable(false).build();
+        } else {
+          return null;
+        }
+      });
+      futures.add(exceptionally);
+    }
+    return futures.stream().map(CompletableFuture::join).filter(Objects::nonNull)
+        .collect(Collectors.toList());
+  }
+
+  private void loadInternal(long blockId, long blockSize, long mountId, String ufsPath, String tag,
+      OptionalInt bandwidth, long offsetInUfs) throws WorkerOutOfSpaceException, IOException {
+    if (bandwidth.isPresent()) {
+      UfsIOManager manager = mUnderFileSystemBlockStore.getOrAddUfsIOManager(mountId);
+      manager.setQuotaInMB(tag, bandwidth.getAsInt());
```

Comment:
it's cumbersome to set for every operation, consider separate into two rpc so we can set once for one load command